inference-engine/src/mkldnn_plugin/nodes/mkldnn_shuffle_channels_node.cpp

```diff
@@ -25,57 +25,35 @@ using namespace mkldnn::impl::cpu::x64;
 
 bool MKLDNNShuffleChannelsNode::isSupportedOperation(const std::shared_ptr<const ngraph::Node>& op, std::string& errorMessage) noexcept {
     try {
-        if (isDynamicNgraphNode(op)) {
-            errorMessage = "Doesn't support op with dynamic shapes";
-            return false;
-        }
-        const auto shuffleChannels = std::dynamic_pointer_cast<const ngraph::op::v0::ShuffleChannels>(op);
+        auto shuffleChannels = ov::as_type_ptr<const ngraph::op::v0::ShuffleChannels>(op);
         if (!shuffleChannels) {
             errorMessage = "Only opset1 ShuffleChannels operation is supported";
             return false;
         }
-        auto shapeSC = shuffleChannels->get_input_shape(0);
-        auto rankSC = shapeSC.size();
-        auto axisSC = shuffleChannels->get_axis();
-        auto groupSC = shuffleChannels->get_group();
-        if (axisSC < 0)
```

Comment:
minor
looks like we can leave check on `axisSC` and `groupSC` since necessary to know attributes value and data rank